corehq/apps/userreports/reports/specs.py

```diff
@@ -73,6 +88,18 @@ def get_column_ids(self):
     def get_column_config(self, data_source_config, lang):
         raise NotImplementedError('subclasses must override this')
 
+    def aggregations(self, data_source_config, lang):
```

Comment:
Not convinced that having these here is the best implementation, but it is better than if/else statements in `get_data`

SQL works because it uses the `sqlagg` library and we directly import columns from there. and it works off of the existing sql reporting framework. Decided this was a good compromise over building up a similar library/es reporting framework